### ISSUE TITLE
Please change list of the streets instead of typing

### ISSUE THREAD

**Issue by BrackoNe** [IpswichMapper/SwiftAddress#30]
It is great that you support autocomplete of the street names, but it is not practical in the reality.
Since it is the same module, here're two things:

- It would be more useful if you show the list of the street names that you get from RevGC, instead of typing street names every time. It can be hard on sunlight, more chance to make an error, but leave the last option to type manually.
- Doesn't remember the last entered street name, so if it is a wrong name, you have to type every single time a street name

**Comment by IpswichMapper** [comment]
What is RevGC?

**Comment by BrackoNe** [comment]
reverse geocoding

**Comment by IpswichMapper** [comment]
> Doesn't remember the last entered street name, 

Yes, I need to add this. This feature existed before but I removed it when I added reverse geocoding. I'll add it back again with a checkbox to choose between reverse geocoding or remembering last street name

**Comment by BrackoNe** [comment]
Sounds great.

**Comment by IpswichMapper** [comment]
> It would be more useful if you show the list of the street names that you get from RevGC, instead of typing street names every time. It can be hard on sunlight, more chance to make an error, but leave the last option to type manually.

Done. List is now shown by default.

> Doesn't remember the last entered street name, so if it is a wrong name, you have to type every single time a street name

Done. Added an option to remember the last input street name. This obiously disabled reverse geocoding however.

NOTE: Last added streetname is automatically rememebered if you turn off your internet.

**Comment by BrackoNe** [comment]
When new build is planned to test this feature?

Thanks for all your great work,
Nemanja

**Comment by IpswichMapper** [comment]
Sorry for the late response. I wanted to release it last weekend, but after adding the "download housenumbers from OSM" feature my datavase has become a mess, and I need to plan how to fix it. So, right now, I don't know.

My hope is to iron out the app completely so that can be released on app stores, so there is quite a but of work to do.
20 Apr 2021, 10:18 by ***@***.***:

**Comment by BrackoNe** [comment]
Sure, no problem. Thanks for the reply.
